app/assets/stylesheets/users.scss

```diff
@@ -78,3 +78,7 @@ label{
 .hidden-label{
 	display:none;
 }
+
+#edit-button {
+	display: none;
```

Comment:
I think it may help to set the `visibility` property for elements we want to hide. There's more detail in this [stackoverflow](https://stackoverflow.com/questions/20598568/how-to-display-and-hide-a-div-with-css#24055535)